src/main/java/org/humancellatlas/ingest/security/SecurityConfig.java

```diff
@@ -13,38 +16,50 @@
 import org.springframework.security.web.util.matcher.AntPathRequestMatcher;
 
 import javax.servlet.http.HttpServletRequest;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
-import java.util.Optional;
+import java.util.*;
 import java.util.stream.Stream;
 
 import static java.util.stream.Collectors.toList;
 import static org.humancellatlas.ingest.security.ElixirConfig.ELIXIR;
 import static org.humancellatlas.ingest.security.GcpConfig.GCP;
-import static org.humancellatlas.ingest.security.Role.GUEST;
-import static org.humancellatlas.ingest.security.Role.WRANGLER;
+import static org.humancellatlas.ingest.security.Role.*;
 import static org.springframework.http.HttpMethod.*;
 
 @EnableWebSecurity
 @Configuration
 public class SecurityConfig extends WebSecurityConfigurerAdapter {
-
-    private static final String FORWARDED_FOR = "X-Forwarded-For";
+    private static final String FORWARDED_HOST = "x-forwarded-host";
 
     private static final List<AntPathRequestMatcher> SECURED_ANT_PATHS;
+
     static {
         List<AntPathRequestMatcher> antPathMatchers = new ArrayList<>();
-        antPathMatchers.addAll(defineAntPathMatchers(GET, "/user/**"));
-        antPathMatchers.addAll(defineAntPathMatchers(PATCH, "/**"));
-        antPathMatchers.addAll(defineAntPathMatchers(PUT, "/**"));
-        antPathMatchers.addAll(defineAntPathMatchers(POST, "/messaging/**", "/files**", "/biomaterials**", "/protocols**", "/processes**",
-                "/files**", "/bundleManifests**"));
+        antPathMatchers.addAll(defineAntPathMatchers(POST, "/**"));
         SECURED_ANT_PATHS = Collections.unmodifiableList(antPathMatchers);
     }
 
+    private static final List<AntPathRequestMatcher> SECURED_WRANGLER_ANT_PATHS;
+
+    static {
```

Comment:
I think it would be cleaner and more readable/declarative to do something like:

```
private static final List<AntPathRequestMatcher> SECURED_WRANGLER_ANT_PATHS = setupWranglerAntPaths();
```

... where `setupWranglerAntPaths()` is a private static method that returns the configured `antPathMatchers` list